docs/Testing.md

```diff
@@ -148,3 +148,19 @@ tap.test('GET `/` route', async (t) => {
   t.deepEqual(response.body, { hello: 'world' })
 })
 ```
+
+### How to inspect `fastify.js` while testing
```

Comment:
I think this should be named "How to inspect tap tests" instead.
This has nothing related to fastify, but I understand why it's good to have this info here.